### ISSUE TITLE
Suggestion: PascalCase property names

### ISSUE THREAD

**Issue by dgvives** [meJevin/Cryptlex.Net#17]
As per Microsoft C# coding conventions, PascalCase property names https://docs.microsoft.com/en-us/dotnet/csharp/fundamentals/coding-style/coding-conventions

This is a breaking change but it may be a good moment to do it before the use of this library explodes

```csharp
namespace Cryptlex.Net.Entities
{
    public class Activation
    {
        public string? id { get; set; }
        public DateTime? createdAt { get; set; }
        public DateTime? updatedAt { get; set; }
        public string? os { get; set; } // "windows" "linux" "macos" "android" "ios"
        public string? osVersion { get; set; }
        public string? hostname { get; set; }
        public Location location { get; set; }
        public string? vmName { get; set; }
        public bool container { get; set; }
        public bool offline { get; set; }
        public string? appVersion { get; set; }
        public string? productId { get; set; }
        public DateTime? expiresAt { get; set; }
        public List<ActivationMetadata>? metadata { get; set; }
        public string? licenseId { get; set; }
        public DateTime? lastSyncedAt { get; set; }
        public DateTime? leaseExpiresAt { get; set; }
        public List<ActivationMeterAttribute>? meterAttributes { get; set; }
    }
}
```

would be:

```csharp
namespace Cryptlex.Net.Entities
{
    public class Activation
    {
        public string? Id { get; set; }
        public DateTime? CreatedAt { get; set; }
        public DateTime? UpdatedAt { get; set; }
        public string? Os { get; set; } // "windows" "linux" "macos" "android" "ios"
        public string? OsVersion { get; set; }
        public string? Hostname { get; set; }
        public Location Location { get; set; }
        public string? VmName { get; set; }
        public bool Container { get; set; }
        public bool Offline { get; set; }
        public string? AppVersion { get; set; }
        public string? ProductId { get; set; }
        public DateTime? ExpiresAt { get; set; }
        public List<ActivationMetadata>? Metadata { get; set; }
        public string? LicenseId { get; set; }
        public DateTime? LastSyncedAt { get; set; }
        public DateTime? LeaseExpiresAt { get; set; }
        public List<ActivationMeterAttribute>? MeterAttributes { get; set; }
    }
}
```

**Comment by meJevin** [comment]
For sure! I'll get on it this weekend 🚀

**Comment by meJevin** [comment]
closed via #21
